### DISCUSSION TITLE
403 error when Migration to 3.3.4

### DISCUSSION THREAD

**Discussion by DPain** [Netflix/conductor#2592] [Q&A]
Hi, I've been working on migrating Conductor 2.30.0 -> Conductor 3.3.4.
I've been able to compile the conductor-server spring boot application and got it running with the mysql-persistence module. (`conductor.db.type= mysql` in `application.properties`) I was able to successfully call the REST endpoint to view existing task definitions and workflow definitions, but I cannot successfully `POST` to `/api/workflow/{workflow_name}` to start those workflows anymore. 

Whenever I do that, I get a 403 Forbidden error.
But as seen here I'm pretty sure I'm calling the correct endpoint: https://github.com/Netflix/conductor/blob/v3.3.4/rest/src/main/java/com/netflix/conductor/rest/controllers/WorkflowResource.java

Problem is, I don't see anywhere in Conductor's source code that returns a 403 error code.

Maybe I might be missing some modules when compiling **conductor-server**? (Using Maven over gradle due to specific requirements.)
The dependencies specifically defined in my `pom.xml` are the following:
- mysql-connector-java (for mysql persistence driver)
- conductor-mysql-persistence
- conductor-rest (to enable rest endpoint)
- conductor-core
- conductor-common
- ~~conductor-grpc-server~~ Removed as it was not needed.
- conductor-contribs

Thanks in advance!

**Comment by aravindanr** [comment]
Conductor does not have any authentication or authorization. Are you building Conductor as part of another Spring Boot application? if you are looking to use rest, you probably should be using `conductor-server` instead of `conductor-grpc-server`.

**Reply by DPain** [reply]
Yes, I'm basically building conductor-server, but using maven instead of gradle. In the process of trying to get the build script working, but seems like I'm missing (or adding too much) dependencies in the pom file haha

I am aware that Conductor has no authentication/authorization which raised me the question why I was getting a 403 error. I'll look further into it and try to separate the issue if it's a spring issue or if it's my maven issue. I did disable all sorts of Spring authentication that I have, but maybe I made a mistake somewhere :)
